src/platform/not_linux.rs

```diff
@@ -30,3 +35,22 @@ impl ClipboardProvider for clipboard_macos::Clipboard {
         self.read()
     }
 }
+
+
+#[cfg(target_os = "ios")]
+mod clipboard_ios {
+    use std::error::Error;
+    pub struct Clipboard;
+    impl Clipboard {
+        pub fn new() -> Result<Clipboard, Box<dyn Error>> {
+            Ok(Self)
+        }
+    }
+}
+
+#[cfg(target_os = "ios")]
+impl ClipboardProvider for clipboard_ios::Clipboard {
+    fn read(&self) -> Result<String, Box<dyn Error>> {
+        unimplemented!();
```

Comment:
Would it be better to return an `Error` here instead of panicking?

I think we could also put this implementation inside the module above.